### ISSUE TITLE
Proper keyboard focus boundary is not visible on the ‘Copy to clipboard’ button.

### ISSUE THREAD

**Issue by msftedad** [docsifyjs/docsify#2307]
#### Steps to reproduce

1. Open the URL: [PWABuilder - Packaging for the Google Play Store](https://docs.pwabuilder.com/#/builder/android) in browser.
2. Navigate to ‘Copy to clipboard’ button through tab key.
3. Observe the issue.

#### Current behaviour
Keyboard focus boundary is not visible on the ‘Copy to clipboard’ button present at the page.

**Attachments:**

<img width="960" alt="Proper keyboard focus boundary is not visible on the ‘Copy to clipboard’ button" src="https://github.com/docsifyjs/docsify/assets/93735775/b5dabdd1-3bde-4fc2-881a-90ca66adf83c">
<img width="959" alt="Proper keyboard focus boundary is not visible on the ‘Copy to clipboard’ button1" src="https://github.com/docsifyjs/docsify/assets/93735775/45d735f3-a429-4ab6-a058-4f821f8d5f87">

#### Expected behaviour

Proper keyboard focus boundary should be visible on the ‘Copy to clipboard’ button.

#### Other relevant information

- Docsify version: 4.4.4

- [ X] Bug still occurs when all/other plugins are disabled?

- Docsify plugins (if the bug happens when plugins enabled, please try to isolate the issue): NA

#### Please create a reproducible sandbox
[PWABuilder - Packaging for the Google Play Store](https://docs.pwabuilder.com/#/builder/android)

#### Mention the docsify version in which this bug was not present (if any)
NA
